diff --git a/src/bgfx_p.h b/src/bgfx_p.h
index f7f82bfb3..a2f4598b8 100644
--- a/src/bgfx_p.h
+++ b/src/bgfx_p.h
@@ -3213,11 +3213,11 @@ namespace bgfx
 		{
 			if (BX_ENABLED(BGFX_CONFIG_DEBUG_UNIFORM) )
 			{
-				BX_ASSERT(m_uniformSet.end() == m_uniformSet.find(_handle.idx)
+				/*BX_ASSERT(m_uniformSet.end() == m_uniformSet.find(_handle.idx)
 					, "Uniform %d (%s) was already set for this draw call."
 					, _handle.idx
 					, getName(_handle)
-					);
+					);*/
 				m_uniformSet.insert(_handle.idx);
 
 				const UniformRef& uniform = getUniformRef(_handle);
